index.html

```diff
@@ -0,0 +1,84 @@
+<html>
+<head>
+    <title>OfekTwitter</title>
+    </meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico">
+</head>
+<body >
+<div class= "topnav">
+    <img src="images/ofek.png">
+    <p>
+        OfekTwitter
+    </p>
+    <div class="topnav-right">
+        <a>Users</a>
+        <a>Sign up</a>
+        <a>Sign in</a>
+    </div>
+</div>
+<div class="twitter-body">
+    <div class="twitter-dashboard-left">
+        <div class="twitter-filter-section">
+            <p>Filtered by </p>
+            <input type="text" placeholder="user name">
+            <input type="button" value="ok">
+        </div>
+        <div class="twitter-flex-container-dashboard">
+            <div class="twitter-person">
+                <img src="images/img_avatar.png"> 
+                <input type="button" value="follow">
```

Comment:
Why the input tag? Why not use button tag?

Reply:
No special reason